src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/TestBindCallGen.generated.txt

```diff
@@ -0,0 +1,122 @@
+// <auto-generated/>
+#nullable enable
+
+using System.Linq;
+
+internal static class GeneratedConfigurationBinder
+{
+    public static void Bind(this global::Microsoft.Extensions.Configuration.IConfiguration configuration, Program.MyClass obj) => BindCore(configuration, ref obj);
+
+    private static void BindCore(global::Microsoft.Extensions.Configuration.IConfiguration configuration, ref Program.MyClass obj)
+    {
+        if (obj is null)
+        {
+            throw new global::System.ArgumentNullException(nameof(obj));
+        }
+
+        if (configuration["MyString"] is string stringValue0)
+        {
+            obj.MyString = stringValue0;
+        }
+
+        if (configuration["MyInt"] is string stringValue1)
+        {
+            obj.MyInt = int.Parse(stringValue1);
```

Comment:
This will be using the current culture to parse.  Is that intended / desirable?  It will also throw if the parsing fails... again, intended?

Reply:
No, it isn't desirable. See https://github.com/dotnet/runtime/pull/82179#discussion_r1131513910.

From @layomia:

> I plan to address this with high priority in a follow-up PR (https://github.com/dotnet/runtime/issues/79527 -"By next preview", in this case preview 3 if the current PR is merged soon).